src/test/java/org/folio/consortia/controller/TenantControllerTest.java

```diff
@@ -347,4 +372,50 @@ void shouldThrownHasActiveAffiliationExceptionWhileDeletingTenant(String content
         status().is4xxClientError(),
         jsonPath("$.errors[0].code", is("VALIDATION_ERROR")));
   }
+
+  @Test
+  void syncPrimaryAffiliations() throws Exception {
+    var headers = defaultHeaders();
+    var consortiumId = UUID.randomUUID();
+    var tenantId = "ABC1";
+
+    this.mockMvc
+      .perform(post(String.format(SYNC_PRIMARY_AFFILIATIONS_URL, consortiumId, tenantId)).headers(headers))
+      .andExpectAll(status().isNoContent());
+
+    //verify(syncPrimaryAffiliationClient, timeout(3000)).primaryAffiliation(any(), anyString(),anyString());
+    //verify(syncPrimaryAffiliationClient, times(1)).primaryAffiliation(any(), anyString(),anyString());
```

Comment:
commented code should be un-commented or deleted, the same in another method of this class